docs/kb/rules/exclude_rule_for_certain_filetypes.md

```diff
@@ -0,0 +1,100 @@
+# How to exclude certain file types for a particular rule
+
+The nature of a certain filetype can generate numerouse false positives and hold up your triage process.  How do you achieve a selective middle ground: continue including the file type to scan against other rules and, at the same time, deflective copious amounts of time triaging false positives?  
```

Comment:
```suggestion
Certain filetypes can generate numerous false positives and hold up your triage process. This document helps you achieve a selective middle ground:
* Continue to include the file type to scan with other rules
* Reduce copious amounts of time triaging false positives
```

[Users tend to skim](https://www.nngroup.com/articles/how-users-read-on-the-web/). I just reduced some words and made sure the value of your document is clear to the user. 

We use the Google Documentation Style Guide and they use a tone that is in between formal (as in textbook-level formal) and conversational (as in blog posts). You can check out [Google Style Guide Voice and Tone](https://developers.google.com/style/tone) but don't worry about having to nail the tone and style, you'll get the hang of it as you write more.